src/components/Dashboard.js

```diff
@@ -18,19 +18,269 @@ import Tooltip from '@material-ui/core/Tooltip';
+                <CardContent>
+                  <Typography variant="h5" component="div">
+                    {event.name}<br />
+                  </Typography>
+                  <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                    Date of Event: {new Date(new Date(event.date).getTime() - (5 * 60 * 60 * 1000)).toLocaleDateString()}<br />
+                  </Typography>
+                  <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                    Location of Event: {event.location}<br />
+                  </Typography>
+                </CardContent>
+              </Card>
+            ))}
+            <Link to={`/events`}>
+              <Typography
+                variant="h6"
+                align="left"
+                gutterBottom
+                component="div">
+                View all events
+              </Typography>
+            </Link>
+          </Grid>
+          <Grid item xs={8} md={4}>
+            <Typography
+              variant="h5"
+              align="center"
+              gutterBottom
+              component="div">
+              Popular Forums
+            </Typography>
+            {forums.map((forum) => (
+              <Card style={{ width: '200 px', marginBottom: '20px' }} key={forum.id}>
+                <CardContent>
+                  <Link to={`/forum/${forum.id}`} target="_blank">
+                    <Typography variant="h5" component="div">
+                      {forum.forumTitle}<br />
+                    </Typography>
+                  </Link>
+                  <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                    {forum.description}<br />
+                  </Typography>
+                </CardContent>
+              </Card>
+            ))}
+            <Link to={`/forums`}>
+              <Typography
+                variant="h6"
+                align="left"
+                gutterBottom
+                component="div">
+                View all forums
+              </Typography>
+            </Link>
+          </Grid>
+          <Grid item xs={8} md={4}>
+            <Typography
+              variant="h5"
+              align="center"
+              gutterBottom
+              component="div">
+              Your RSVPed Events
+            </Typography>
+            {myEvents.map((myEvent) => (
+              <Card style={{ width: '200 px', marginBottom: '20px' }} key={myEvent.id}>
+                <CardContent>
+                  <Typography variant="h5" component="div">
+                    {myEvent.name}<br />
+                  </Typography>
+                  <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                    Date of Event: {new Date(new Date(myEvent.date).getTime() - (5 * 60 * 60 * 1000)).toLocaleDateString()}<br />
+                  </Typography>
+                  <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                    Location of Event: {myEvent.location}<br />
```

Comment:
"Location" instead of "Location of Event"

Reply:
Can you update this "Location" and "Date" change to the RSVP'd list of events as well? Other than that everything looks great!